### DISCUSSION TITLE
Has anyone connected to Tableau?

### DISCUSSION THREAD

**Discussion by nbanderso** [mcguirepr89/BirdNET-Pi#338] [General]
Hi all.  

I am trying to find a way to make a direct connection from my birdnet-pi database to a tableau dashboard.  Has anyone done this?  If so, can you point me in the right direction for making that connection?  

I'm hoping to make some cool visualizations of migration patterns in my area.  

Thank you in advance!

**Comment by CaiusX** [comment]
Hi @nbanderso - good to see another data viz junkie. 

My hack is to download the birdsdb.txt and pull in as a csv. 

If you come up with good results maybe we can include in the streamlit interface. 

Best
AP

**Reply by nbanderso** [reply]
Thank you!  I'll definitely be using this one but I was hoping for a more passive solution.  

My long term goal is to track migration patterns and having the data automatically updated on a server that can be read by Tableau (or Pwer BI) would make it much easier!  

Thanks again.

**Reply by ehpersonal38** [reply]
I dont know anything about Tableau, but it looks like it supports connecting to an sqlite database, which is what BirdNET-Pi uses.

https://help.tableau.com/current/pro/desktop/en-us/odbc_customize.htm

**Reply by nbanderso** [reply]
Just to close the loop, I was able to connect directly to the SQLite database in Tableau by following the instructions in the link above.  

The only other step that I had to complete was installing Samba on the raspberry pi so that I could access the birds.db file on my windows machine (where tableau is installed).  

Thanks again @CaiusX  and @ehpersonal38

**Reply by CaiusX** [reply]
Great- thanks for the feedback. It will be good to see what you come up with.
